src/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.AutoMLFeaturizers.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.Featurizers</IncludeInPackage>
```

Comment:
`IncludeInPackage` needs to match the name of the NuGet package. Or else this assembly won't be included in the right NuGet package.

Reply:
I am not sure why it is showing up like that. Last night I pushed a fix that had that change to the correct name. It looks like somehow the branch in my repo and this PR aren't syncing up correctly. The number of commits this PR shows is not the same as the number of commits in my branch itself? I will rebase and see if that fixes the issue.